specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2023-06-01-preview/agfood.json

```diff
@@ -1876,19 +1876,16 @@
       "allOf": [
         {
           "$ref": "#/definitions/AuthCredentials"
-        },
-        {
-          "required": [
-            "apiKey"
-          ],
-          "type": "object",
-          "properties": {
-            "apiKey": {
-              "$ref": "#/definitions/KeyVaultProperties"
-            }
-          }
         }
       ],
+      "required": [
+        "apiKey"
+      ],
+      "properties": {
+        "apiKey": {
+          "$ref": "#/definitions/KeyVaultProperties"
+        }
+      },
```

Comment:
The old swagger had these in wrong place, just curious, is this already deployed? And your service is generating these required data correctly?

Reply:
Yes, old swagger is deployed, and our service will generate the new swagger in it.